CONTRIBUTING.md

```diff
@@ -56,9 +56,11 @@ Other notes:
 
 - Looking for a good place to start contributing? How about checking out some
   [good first issues](https://github.com/cosmos/cosmos-sdk/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22)
-- Please make sure to use `gofmt` before every commit - the easiest way to do
-  this is have your editor run it for you upon saving a file. Additionally
-  please ensure that your code is lint compliant by running `make lint`
+- Please make sure to run `make format` before every commit - the easiest way
+  to do this is have your editor run it for you upon saving a file. Additionally
+  please ensure that your code is lint compliant by running `golangci-lint run`.
```

Comment:
Is it possible/desirable to run `golanci-lint` in the pre-commit hook too?

Reply:
I thought abour it, and one could even argue why don't we incorporate `make test-unit` too.
My concern about running `golangci-lint` is shown in the following screenshot:

![image](https://user-images.githubusercontent.com/229356/79909174-cbb7ea00-841c-11ea-9491-15572297dcc4.png)

With some setups, it might cause developer's machine to hang.

Reply:
@aaronc please take a look:

https://github.com/cosmos/cosmos-sdk/pull/6041/commits/a3e1fa8564fe5f0802e617c8c3a2766a34893d75

Reply:
It slows down `git commit` quite a bit